hack/make/manifests/openshift.mk

```diff
@@ -2,8 +2,13 @@
 manifests/openshift/csi:
```

Comment:
unrelated, but why does the `Common` folder start with an uppercase letter?

Reply:
no clue; too afraid to change it now 😅 (maybe related to GKE marketplace 🤷 )